### DISCUSSION TITLE
q-table inside a q-tabs

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#9384] [CLI - PWA mode]
Hello Friends ,

I need some help on a problem am facing. am using 2 q-tabs  one contains a q-table in which i have a radio button and second contain some input fields. when i select a row in the table and i move to the second tab and i move back to the first tab the row i selected is cleared.

Note that i dont want to use keep-alive

Can someone please help 

Thanks 
[q-table.txt](https://github.com/quasarframework/quasar/files/6527195/q-table.txt)
![tabs](https://user-images.githubusercontent.com/84211129/119241418-3333f200-bb67-11eb-966b-345d49d57e1a.png)

**Comment by ontwikkelfabriek** [comment]
@akashramtohul4 
If you don't want to use keep alive, you could store/retrieve the state( vuex would be a good fit) of the qtable in combination with Vue lifecycle hooks to store/retrieve the state of the table.

**Reply by akashramtohul4** [reply]
@ontwikkelfabriek  
How can i get the row i selected? its a custom table
